lib/convertor.js

```diff
@@ -252,9 +252,11 @@ class MongoToKnex {
                             if (negateGroup) {
                                 statementOp = compOps.$in;
                             } else {
-                                statementOp = isNegationOp(statement.operator)
-                                    ? compOps.$nin
-                                    : compOps.$in;
+                                if (isNegationOp(statement.operator)) {
+                                    statementOp = compOps.$nin;
+                                } else {
+                                    statementOp = compOps[statement.operator];
+                                }
                             }
                             const statementValue = !_.isArray(statement.value) ? [statement.value] : statement.value;
```

Comment:
This part broke SQLite because it couldn't compare array values with `=` operator (MySQL doesn't care if its `slug = 'a'` or `slug = ('a')`)